### ISSUE TITLE
Clarification on azimuthal and polar angle bounds

### ISSUE THREAD

**Issue by matthewturk** [spherical-volume-rendering/svr-algorithm#207]
Hi!  I was working on some of the previous work when I ran into something that I was unsure of.  In the `test_svr.cpp` file, I see lots of areas where the sphere bounds are set to 0 .. 2*pi for both azimuthal and polar angles.  My understanding is that we should usually be setting them to 0..pi for polar and 0..2pi for azimuth.

Is the code set up to address this somewhere internally?  Or does it need to be modified somehow to take this into account?

**Comment by cgyurgyik** [comment]
I believe I just set it up in a way such that we treat the azimuthal bound as a circle on the Z-axis, so that `[0, 2pi]` encompasses the entire sphere while `[0, pi]` might only encompass a portion of it. This could probably be refactored to use `[0, pi]`. I can't think of a reason why not.

I don't believe we ever modify ourselves; we simply treat it as `[0, 2pi]`.

Like I said earlier, the bounds issue was a bit rushed and definitely far from perfect. This was probably my fault, and I should've taken a step back. But then school deadlines happened.

**Comment by matthewturk** [comment]
OK -- we could delay talking through this further until you're off your
internship, if that works better for you!

**Comment by cgyurgyik** [comment]
Yep, I'll probably tag along and read what you post here every once in awhile. Hopefully since next week is my last, its a bit calmer.

**Comment by ak-2485** [comment]
Yes, so this is kind of weird, and perhaps it is worse and than weird and just wrong: we aren't thinking about a coordinate system - we wanted to avoid a coordinate system as much as possible - and just figure out a simple way to count voxels. The way we did this was to break up the interval [0,2pi] by the number of desired sections for each angular region. I imagine that we will want to switch to the standard [0,pi] for the azimuthal angle/angle of inclination; this would only affect the initialization routine which determines the initial voxel of the ray.

**Comment by cgyurgyik** [comment]
Will change to [0, pi] once #208 is reviewed and submitted.
